README.md

```diff
@@ -167,19 +193,27 @@ Listening on localhost:4567, CTRL+C to stop
 
 ### Troubleshooting
 
-**Problem:** When you try to run the app with `rake spec` or `ruby app.rb` and
-receive errors like:
+**Problem:** *Running the app with `rake spec` or `ruby app.rb` I receive
```

Comment:
This should probably be "Running the app with `rake spec` or `ruby app.rb` **you** receive errors like"